compiler/il/OMRBlock.cpp

```diff
@@ -1226,7 +1228,11 @@ static TR::SymbolReference * createSymRefForNode(TR::Compilation *comp, TR::Reso
 
    if (!symRef)
       {
-      symRef = comp->getSymRefTab()->createTemporary(methodSymbol, value->getDataType());
+      symRef = comp->getSymRefTab()->createTemporary(methodSymbol, value->getDataType(), false,
+#ifdef J9_PROJECT_SPECIFIC
+                                                      value->getType().isBCD() ? value->getSize() :
+#endif
```

Comment:
Just noting we already have an existing `#ifdef J9_PROJECT_SPECIFIC` in this function in line 1140 above which does the same thing as here, so this is not rally introducing a "new" `J9_PROJECT_SPECIFIC` occurrence.